src/sentry/integrations/vsts/issues.py

```diff
@@ -218,4 +218,6 @@ def get_done_states(self, project):
         return done_states
 
     def get_issue_display_name(self, external_issue):
+        if external_issue.metadata is None:
```

Comment:
this was breaking for me locally. in prod, i think there is one issue this will happen for